webrender/res/debug_color.vs.glsl

```diff
@@ -2,6 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+in vec4 aColorRectTL;
```

Comment:
Yep, I have it on my list. Clearly, TL/RectTop suffices are misleading with this change